pkg/actors/actors.go

```diff
@@ -456,9 +456,8 @@ func (a *actorsRuntime) callRemoteActorWithRetry(
 				Disposer: resiliency.DisposerCloser[*invokev1.InvokeMethodResponse],
 			},
 		)
-		attempts := atomic.Int32{}
 		return policyRunner(func(ctx context.Context) (*invokev1.InvokeMethodResponse, error) {
-			attempt := attempts.Add(1)
+			attempt := resiliency.GetAttempt(ctx)
```

Comment:
I think this line should be after the function call....

Reply:
It doesn't matter. The attempt is set in the context so the value would be the same.

Reply:
Only after the function is run would the context be updated?

Reply:
The context is set before the function runs. So the attempt is the same throughout the policy function

Also, changing this to increment `attempts` after (which would be done in the `resiliency` pkg) would cause incorrect numbers in case the function call takes longer than the allowed timeout (a sort of race condition)